From 6f7bf9b8a75f106e24213ffc33d948721aba1a3f Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Sun, 18 Sep 2016 20:38:21 +0800
Subject: [PATCH 0986/2241] Revert "arm64: dts: a3700: enable hw iocc for all
 io units"

- As a generic feature for all flavors of Armada-3700 rev A0,
  HW-IOCC should be declared in SoC dtsi - armada-37xx.dtsi.
  It is supposed to be fixed by late patches.
- This reverts commit 19153f1a075d ("arm64: dts: a3700: enable
  hw iocc for all io units").

Change-Id: I1898cfb0c034cc35f920cc6a9e4edb0a5dc93c2a
Reviewed-on: http://vgitil04.il.marvell.com:8080/32832
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Tested-by: Wilson Ding <dingwei@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32920
---
 arch/arm64/boot/dts/marvell/armada-371x.dtsi | 34 ----------------------------
 arch/arm64/boot/dts/marvell/armada-372x.dtsi | 33 ---------------------------
 2 files changed, 67 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-371x.dtsi b/arch/arm64/boot/dts/marvell/armada-371x.dtsi
index 486ed82..c9e5325 100644
--- a/arch/arm64/boot/dts/marvell/armada-371x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-371x.dtsi
@@ -50,38 +50,4 @@
 / {
 	model = "Marvell Armada 3710 SoC";
 	compatible = "marvell,armada3710", "marvell,armada3700";
-	soc {
-		internal-regs {
-			eth0: ethernet@30000 {
-				   dma-coherent;
-			};
-			eth1: ethernet@40000 {
-				dma-coherent;
-			};
-			usb3@58000 {
-				dma-coherent;
-			};
-			usb@5e000 {
-				dma-coherent;
-			};
-			xor@60900 {
-				dma-coherent;
-			};
-			sdhci1: sdhci@d0000 {
-				dma-coherent;
-			};
-
-			sdhci0: sdhci@d8000 {
-				dma-coherent;
-			};
-			sata@e0000 {
-				dma-coherent;
-			};
-			pcie-controller {
-				pcie@1,0 {
-					dma-coherent;
-				};
-			};
-		};
-	};
 };
diff --git a/arch/arm64/boot/dts/marvell/armada-372x.dtsi b/arch/arm64/boot/dts/marvell/armada-372x.dtsi
index fa2fd89..f292a00 100644
--- a/arch/arm64/boot/dts/marvell/armada-372x.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-372x.dtsi
@@ -59,38 +59,5 @@
 			enable-method = "psci";
 		};
 	};
-	soc {
-		internal-regs {
-			eth0: ethernet@30000 {
-				   dma-coherent;
-			};
-			eth1: ethernet@40000 {
-				dma-coherent;
-			};
-			usb3@58000 {
-				dma-coherent;
-			};
-			usb@5e000 {
-				dma-coherent;
-			};
-			xor@60900 {
-				dma-coherent;
-			};
-			sdhci1: sdhci@d0000 {
-				dma-coherent;
-			};
 
-			sdhci0: sdhci@d8000 {
-				dma-coherent;
-			};
-			sata@e0000 {
-				dma-coherent;
-			};
-			pcie-controller {
-				pcie@1,0 {
-					dma-coherent;
-				};
-			};
-		};
-	};
 };
-- 
2.7.4

